db/migrate/20210531154029_devise_token_auth_create_users.rb

```diff
@@ -36,6 +36,8 @@ def change
       t.timestamps
 
       t.integer :gender
+      t.string :first_name, default: ''
+      t.string :last_name, default: ''
```

Comment:
It's not a good practice to edit an existing migration, if we want to make a change on any existing model we should create a new migration.

If a migration has already been run on production, you'll need to make a rollback and then run the migration again.